test/unit/lib/static/components/section/body/meta-info.js

```diff
@@ -34,4 +34,23 @@ describe('<MetaInfo />', () => {
             assert.equal(node.text(), expectedMetaInfo[i]);
         });
     });
+
+    it('should render meta info with non primitive values', () => {
+        const expectedMetaInfo = [
+            'foo1: {"bar":"baz"}',
```

Comment:
There's no spaces after `:`. Is it done purposely?

Reply:
It doesn't matter, `JSON.parse` works both cases.